client/src/pages/board/components/dashboardForms/dialogSections/enums.js

```diff
@@ -0,0 +1,70 @@
+import DescIcon from '@material-ui/icons/ImportContactsTwoTone';
+import DeadlineIcon from '@material-ui/icons/ScheduleOutlined';
+import CommentsIcon from '@material-ui/icons/ForumOutlined';
+import AttachmentIcon from '@material-ui/icons/AttachFileOutlined';
+import ChecklistIcon from '@material-ui/icons/PlaylistAddCheckRounded';
+import TagsIcon from '@material-ui/icons/LabelRounded';
+import {colors} from '@material-ui/core';
+import CardDialogDesc from './CardDialogDesc';
+import CardDialogChecklist from './CardDialogChecklist';
+import CardDialogDeadline from './CardDialogDeadline';
+import CardDialogComments from './CardDialogComments';
+import CardDialogAttachments from './CardDialogAttachments';
+import CardDialogTags from './CardDialogTags';
```

Comment:
Really modular lol. I feel it bloats up the imports section, but it's still good abstraction. I personally would just prefer an Object that would hold each CardDialogSection instead. All preference tbh

Reply:
I feel like importing a lot of stuff is a worth price to pay for separating segments into different files (need to be careful not to overdo it though for sure). I'd say this branch is probably on the edge (if not a bit over) of overdoing it but sometimes things just get out of control :). One thing I like about this is when we make changes to one section, it'll be very clear in the git interface.